io.c

```diff
@@ -15160,39 +15168,52 @@ set_LAST_READ_LINE(VALUE val, ID _x, VALUE *_y)
  *    f.gets        # => "e\n"
  *    f.close
  *
- *  There are two special line separators:
+ *  Or by setting global variable <tt>$/</tt>:
+ *
+ *    f = File.new('t.txt')
+ *    $/ = 'l'
+ *    p f.gets
+ *    p f.gets
+ *    p f.gets
+ *    f.close
+ *
+ *  Output:
+ *
+ *    "First l"
+ *    "ine\nSecond l"
+ *    "ine\n\nFourth l"
+ *
+ *  ===== Special Line Separator Values
  *
- *  - +nil+: The entire stream is read into a single string:
+ *  Each of the {line input methods}[rdoc-ref:IO@Line+Input]
+ *  accepts two special values for parameter +sep+:
+ *
+ *  - +nil+: The entire stream is to be read ("slurped") into a single string:
```

Comment:
Nit: why are we calling it "slurp"? It doesn't seem like a term used anywhere in the docs.

Reply:
The term is well established in other interpreted languages (Perl and Python in particular), where its meaning is "read it all."